src/patternfly/components/Wizard/wizard.scss

```diff
@@ -0,0 +1,470 @@
+  position: relative;
+  display: inline-block;
+  color: var(--pf-c-wizard__nav-link--Color);
+  text-decoration: var(--pf-c-wizard__nav-link--TextDecoration);
+  word-break: break-word;
+
+  // Common step number styles
+  @at-root .pf-c-wizard__toggle-num,
+  &::before {
+    position: absolute;
+    top: 0;
+    left: 0;
+    display: inline-flex;
+    align-items: center;
+    justify-content: center;
+    width: var(--pf-c-wizard__nav-link--before--Width);
+    height: var(--pf-c-wizard__nav-link--before--Height);
+    font-size: var(--pf-c-wizard__nav-link--before--FontSize);
+    line-height: 1;
+    color: var(--pf-c-wizard__nav-link--before--Color);
+    background-color: var(--pf-c-wizard__nav-link--before--BackgroundColor);
+    border-radius: var(--pf-c-wizard__nav-link--before--BorderRadius);
+    transform: var(--pf-c-wizard__nav-link--before--Transform);
+  }
+
+  // Nav step number
+  &::before {
+    content: counter(wizard-nav-count);
+    counter-increment: wizard-nav-count;
+  }
+
+  &:hover,
+  &.pf-m-hover {
+    --pf-c-wizard__nav-link--Color: var(--pf-c-wizard__nav-link--hover--Color);
+  }
+
+  &:focus,
+  &.pf-m-focus {
+    --pf-c-wizard__nav-link--Color: var(--pf-c-wizard__nav-link--focus--Color);
+  }
+
+  &.pf-m-current {
+    --pf-c-wizard__nav-link--Color: var(--pf-c-wizard__nav-link--m-current--Color);
+
+    @at-root .pf-c-wizard__toggle-num,
+    &::before {
+      --pf-c-wizard__nav-link--before--BackgroundColor: var(--pf-c-wizard__nav-link--m-current--before--BackgroundColor);
+      --pf-c-wizard__nav-link--before--Color: var(--pf-c-wizard__nav-link--m-current--before--Color);
+    }
+  }
+
+  &.pf-m-disabled {
+    --pf-c-wizard__nav-link--Color: var(--pf-c-wizard__nav-link--m-disabled--Color);
+
+    pointer-events: none;
+
+    &::before {
+      --pf-c-wizard__nav-link--before--BackgroundColor: var(--pf-c-wizard__nav-link--m-disabled--before--BackgroundColor);
+      --pf-c-wizard__nav-link--before--Color: var(--pf-c-wizard__nav-link--m-disabled--before--Color);
+    }
+  }
+}
+
+.pf-c-wizard__main {
+  flex-grow: 1;
+  grid-column: 1 / -1;
+  grid-row: 3 / 4;
+  overflow: auto;
+  -webkit-overflow-scrolling: touch;
+
+  @media screen and (min-width: $pf-global--breakpoint--lg) {
+    grid-column: 2 / 3;
+    grid-row: 2 / 3;
+  }
+}
+
+.pf-c-wizard__body {
+  padding: var(--pf-c-wizard__body--PaddingTop) var(--pf-c-wizard__body--PaddingRight) var(--pf-c-wizard__body--PaddingLeft) var(--pf-c-wizard__body--PaddingRight);
+}
+
+.pf-c-wizard__footer {
+  display: flex;
+  flex-wrap: wrap;
+  padding: var(--pf-c-wizard__footer--PaddingTop) var(--pf-c-wizard__footer--PaddingRight) var(--pf-c-wizard__footer--PaddingBottom) var(--pf-c-wizard__footer--PaddingLeft);
+  grid-column: 1 / -1;
+  grid-row: 4 / 5;
+
+  @media screen and (min-width: $pf-global--breakpoint--lg) {
+    grid-column: 2 / 3;
+    grid-row: 3 / 4;
+  }
+
+  > * {
+    margin-bottom: var(--pf-c-wizard__footer--child--MarginBottom);
```

Comment:
Is this needed? Is it for wrapping buttons?

Reply:
Yeah, and the footer's `padding-bottom` is adjusted to account for the button's `margin-bottom